index.html

```diff
@@ -25,187 +25,216 @@
+                    </ul>
+                    <p class="description-Text">A daily selection of privately personalized reads; no accounts or
+                        sign-ups required.</p>
+                    <div class="lang">
+                        <ul class="languages">
+                            <li><button class="languages-List" type="submit">html</button></li>
+                            <li><button class="languages-List" type="submit">css</button></li>
+                            <li><button class="languages-List" type="submit">javascript</button></li>
+                        </ul>
+                    </div>
+                    <button class="btn-Project" type="submit">See Project</button>
                 </div>
-                <button class="btn-Project" type="submit">See Project</button>
-            </div>
             </div>
 
             <div class="works-Change">
                 <div class="desktop-Img-Works flip">
                     <img class="snapshoot" src="images/Snapshoot-Portfolio4.svg" alt="Snapshoot-Image">
                 </div>
                 <div class="desktop-Text-Works">
-                <h2 class="tonic">Multi-Post Stories</h2>
-                <ul class="sub-Points">
-                    <li class="canopy">CANOPY</li>
-                    <li class="sub-Text">Back End Dev</li>
-                    <li class="sub-Text">2015</li>
-                </ul>
-                <p class="description-Text">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
-                <div class="lang"> 
-                <ul class="languages">
-                    <li><button class="languages-List" type="submit">html</button></li>
-                   <li><button class="languages-List" type="submit">css</button></li>
-                    <li><button class="languages-List" type="submit">javascript</button></li>
-                </ul>
+                    <h2 class="tonic">Multi-Post Stories</h2>
+                    <ul class="sub-Points">
+                        <li class="canopy">CANOPY</li>
+                        <li class="sub-Text">Back End Dev</li>
+                        <li class="sub-Text">2015</li>
+                    </ul>
+                    <p class="description-Text">A daily selection of privately personalized reads; no accounts or
+                        sign-ups required.</p>
+                    <div class="lang">
+                        <ul class="languages">
+                            <li><button class="languages-List" type="submit">html</button></li>
+                            <li><button class="languages-List" type="submit">css</button></li>
+                            <li><button class="languages-List" type="submit">javascript</button></li>
+                        </ul>
+                    </div>
+                    <button class="btn-Project" type="submit">See Project</button>
                 </div>
```

Comment:
- [ ] Kindly the `projects` section should be deleted from the HTML file and created by JS dynamically when the page loads, as required in the requirements `You need to update the main page so that the projects section is created dynamically using the information stored in that JavaScript object. Remember, all of the HTML in that section is created when the page loads.`